presto-main/src/main/java/com/facebook/presto/sql/planner/EqualityInference.java

```diff
@@ -364,14 +364,14 @@ private RowExpression normalizeInPredicateToEquality(RowExpression expression)
         /**
          * Provides a convenience Iterable of RowExpression conjuncts which have not been added to the inference
          */
-        public Iterable<RowExpression> nonInferrableConjuncts(RowExpression expression)
+        public Iterable<RowExpression> nonInferableConjuncts(RowExpression expression)
         {
             return filter(extractConjuncts(expression), not(isInferenceCandidate()));
         }
 
-        public static Iterable<RowExpression> nonInferrableConjuncts(Metadata metadata, RowExpression expression)
+        public static Iterable<RowExpression> nonInferableConjuncts(Metadata metadata, RowExpression expression)
```

Comment:
APIs?

Reply:
Some projects worry about API/schema changes, so when I make large PRs (as this one), I try to call out the things that I expect might be the most risky, because I don't want to be seen as a bumbler who isn't concerned with the code I'm changing. (This change here is removing an `r` from `inferable`.)

For this project, if this counts as an API, then I imagine it'd end up mentioned in the `release notes` section of a PR (and I'd likely offer to split it and its brethren into a PR distinct from one that doesn't make such changes).